src/api/MainSolver.cc

```diff
@@ -203,12 +201,12 @@ void MainSolver::printFramesAsQuery(std::ostream & s) const {
 // that structure.
 //
 PTRef MainSolver::rewriteMaxArity(PTRef root) {
-    return ::rewriteMaxArityClassic(logic, root);
+    return opensmt::rewriteMaxArityClassic(logic, root);
```

Comment:
Is the namespace here needed?

Reply:
It is not necessary, but I kept it the way as it was before. Nevertheless, I think it's good to emphasize that a free function from outside is used instead of a member function, since the class defines similar functions.